### PR TITLE
Refactor current spot with constant

### PR DESCRIPTION
**Pull request by AnnieLeonia**

### CODE REVIEW

File: client/src/components/CurrentSpot.vue [hunk 1]
```diff
@@ -30,56 +33,26 @@
 
 <script>
 import Vue from "vue";
-import distanceIcon from "@/assets/distance.png";
-import starIcon from "@/assets/star.png";
+import { mapState, mapMutations } from "vuex";
+import { modalStateQRspot } from "../constans";
```
Comment (VictorWinberg) [inline client/src/components/CurrentSpot.vue:37]:
```suggestion
import { modalStateQRspot } from "@/constans";
```

File: client/src/components/GoogleMaps.vue [hunk 2]
```diff
@@ -50,6 +55,8 @@
 
 <script>
 import Vue from "vue";
+import { mapState } from "vuex";
+import { modalStateQRspot } from "../constans";
```
Comment (VictorWinberg) [inline client/src/components/GoogleMaps.vue:59]:
```suggestion
import { modalStateQRspot } from "@/constans";
```

File: client/src/components/GoogleMaps.vue [hunk 3]
```diff
@@ -151,16 +125,17 @@ export default Vue.extend({
...
-      this.markerSelected = index;
-      this.currentSpot = this.currentMarker;
-      this.showSpotInfo = true;
+    selectQRspot(marker) {
+      this.map.panTo(new google.maps.LatLng(marker.lat, marker.lng));
+      this.$store.commit("qrSpot/setQrSpot", marker);
+      this.$store.commit(
+        "qrSpot/setShowQrSpot",
+        this.modalStateQRspot.SHOW_INFO
+      );
```
Comment (VictorWinberg) [inline client/src/components/GoogleMaps.vue:134]: Use `mapMutations`?

**Review by VictorWinberg — CHANGES_REQUESTED** [review]
Some comments 🥳

File: client/src/components/GoogleMaps.vue (hunk 3 above)
Comment (AnnieLeonia) [reply client/src/components/GoogleMaps.vue]: Can you have another look now and see if is correct?

**Review by VictorWinberg — APPROVED** [review]
:tada:

File: client/src/components/GoogleMaps.vue (hunk 3 above)
Comment (VictorWinberg) [reply client/src/components/GoogleMaps.vue]: :heart:

**Pull request merged**